CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to
 
 ### Fixed
 
+-  ✨(auth) fix empty names from ProConnect #687
```

Comment:
```suggestion
- ✨(auth) fix empty names from ProConnect #687
```

Reply:
We have a dedicated test (test_authentication_getter_new_user_with_email) but it was not broken prior to this PR as prod was. I'm not confident the way we mock is the way ProConnect sends its data. I'm going to check that out.